Flaky gc test

Fails in CI on windows sometimes:

```
ipfs:     .repo.gc
ipfs:       1) should clean up indirectly pinned data after recursive pin removal
ipfs:   1109 passing (18m)
ipfs:   69 pending
ipfs:   1 failing
ipfs:   1) interface-ipfs-core over ipfs-http-client tests
ipfs:        .repo.gc
ipfs:          should clean up indirectly pinned data after recursive pin removal:
ipfs:      AssertionError: expected [ Array(19) ] to not include 'QmYUY5XnbdTH914VW6dfGSvuyMuBXdjaHk2aUKNWQxMMNQ'
ipfs:       at Context.it (C:\Users\travis\build\ipfs\js-ipfs\packages\interface-ipfs-core\src\repo\gc.js:58:377)
ipfs:       at process._tickCallback (internal/process/next_tick.js:68:7)
ipfs: 
```

Over http to go-ipfs:

```
  interface-ipfs-core tests
    .repo.gc
2020-04-27T17:43:29.473Z ipfsd-ctl:daemon:stdout initializing IPFS node at C:\Users\alex\AppData\Local\Temp\go_ipfs_V8bdl8I5cb2u60M8B5gdp
generating 1024-bit RSA keypair...done
peer identity: QmV6rMbYE7712aQxAERMYdZ11bsxrYdcno7dTVKkLkdKkp
to get started, enter:

        ipfs cat /ipfs/QmS4ustL54uo8FzR9455qaxZwuMiUhyvMcX9Ba8nUH4uVv/readme

2020-04-27T17:43:29.473Z ipfsd-ctl:daemon:stderr
2020-04-27T17:43:29.712Z ipfsd-ctl:utils Unable to open api file
2020-04-27T17:43:29.837Z ipfsd-ctl:daemon:stdout Initializing daemon...
go-ipfs version: 0.4.23-
Repo version: 7
System version: amd64/windows
Golang version: go1.13.7

2020-04-27T17:43:29.910Z ipfsd-ctl:daemon:stdout Swarm listening on /ip4/127.0.0.1/tcp/51970

2020-04-27T17:43:29.910Z ipfsd-ctl:daemon:stdout Swarm listening on /p2p-circuit

2020-04-27T17:43:29.920Z ipfsd-ctl:daemon:stdout Swarm announcing /ip4/127.0.0.1/tcp/51970

2020-04-27T17:43:29.920Z ipfsd-ctl:daemon:stdout API server listening on /ip4/127.0.0.1/tcp/51971

2020-04-27T17:43:29.953Z ipfsd-ctl:daemon:stdout WebUI: http://127.0.0.1:51971/webui

2020-04-27T17:43:29.989Z ipfsd-ctl:daemon:stdout Gateway (readonly) server listening on /ip4/127.0.0.1/tcp/51972

2020-04-27T17:43:30.026Z ipfsd-ctl:daemon:stdout Daemon is ready

(node:6316) ExperimentalWarning: Readable[Symbol.asyncIterator] is an experimental feature. This feature could change at any time
      √ should run garbage collection (104ms)
      √ should clean up unpinned data (139ms)
      √ should clean up removed MFS files (124ms)
      √ should clean up block only after unpinned and removed from MFS (185ms)
2020-04-27T17:43:30.814Z ipfsd-ctl:daemon:stderr 18:43:30.814 ERROR blockstore: blockstore.AllKeysChan got err: walk failed: open C:\Users\alex\AppData\Local\Temp\go_ipfs_V8bdl8I5cb2u60M8B5gdp\blocks\du-195716584: The process cannot access the file because it is being used by another process. blockstore.go:222

      1) should clean up indirectly pinned data after recursive pin removal


  4 passing (2s)
  1 failing

  1) interface-ipfs-core tests
       .repo.gc
         should clean up indirectly pinned data after recursive pin removal:
     AssertionError: expected [ Array(4) ] to include 'QmYUY5XnbdTH914VW6dfGSvuyMuBXdjaHk2aUKNWQxMMNQ'
      at Context.it (C:\Users\alex\Documents\Workspaces\ipfs\js-ipfs\packages\interface-ipfs-core\src\repo\gc.js:215:43)
      at process._tickCallback (internal/process/next_tick.js:68:7)

```

--- a/packages/interface-ipfs-core/src/repo/gc.js
+++ b/packages/interface-ipfs-core/src/repo/gc.js
@@ -180,7 +180,7 @@ module.exports = (common, options) => {
       const obj = await new DAGNode(Buffer.from('fruit'), [{
         Name: 'p',
         Hash: dataCid,
-        TSize: addRes[0].size
+        Tsize: addRes[0].size
       }])
 
       // Put the object into IPFS

--- a/packages/ipfs-http-client/src/repo/gc.js
+++ b/packages/ipfs-http-client/src/repo/gc.js
@@ -18,6 +18,6 @@ module.exports = configure(api => {
       }
     })
 
-    yield res.ndjson()
+    yield * res.ndjson()
   }
 })

